ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -147,6 +147,44 @@ debug_pod() {
         oc logs "$pod"
 }
 
+# Wait for all pods of the app label to contain expected message
+#
+# Params:
+#   $1 - app label
+#   $2 - message to be present in the logs
+#   $3 - timeout (60)
+#   $4 - namespace (the current one)
+wait_for_app_pods_message() {
+	local app="$1"
+	local message="$2"
+	local timeout="$3"
+	local namespace="$4"
+	[ -z "$timeout" ] && timeout=60
+	[ -n "$namespace" ] && namespace=" -n $namespace "
+	local pod
+	local pods
+	local i
+	SECONDS=0
+	while :; do
+		pods=$(oc get pods -l app="$app" --no-headers=true $namespace | awk '{print $1}' | xargs)
```

Comment:
No doubt that I have is whether the daemonset is going to schedule N pods at same time or not. In a situation where a node is not ready *yet* then it won't schedule to that node?

Reply:
> No doubt that I have is whether the daemonset is going to schedule N pods at same time or not. In a situation where a node is not ready _yet_ then it won't schedule to that node?

The number of pods doesn't depend on the status of the nodes specifically. It only depends on the selector.
The pods being scheduled on these nodes is another story.

Since the same selector that is used in the daemonset YAML is passed to `oc get pods`, we should expect to get the exact list of pods at some point in time. Not sure it is guaranteed that all pods are created atomically.

Maybe safer to only break when we get the expected number of pods, which can be passed as an argument to this function.

Reply:
Sounds good, I'll update that